lib/kontena/plugin/shell/session.rb

```diff
@@ -27,6 +27,18 @@ def run_command(buf)
         tokens = buf.split(/\s(?=(?:[^"]|"[^"]*")*$)/).map(&:strip)
         runner = Shell.command(tokens.first) || Shell.command(context.first) || Kontena::Plugin::Shell::KontenaCommand
         command = runner.new(context, tokens, self)
+        if fork_supported?
+          execute_with_fork(command)
+        else
+          execute_with_thread(command)
+        end
+      end
+
+      def fork_supported?
+        !Gem.win_platform?
```

Comment:
`Process.respond_to?(:fork)` could be more universal.

> If fork is not usable, Process.respond_to?(:fork) returns false. (http://www.rubydoc.info/stdlib/core/Process.fork)

Reply:
Fixed.